lib/ActivityStreams.js

```diff
@@ -67,6 +68,12 @@ function ActivityStreams(options = {}) {
     preview: false,
   };
 
+  // This lets us know whether or not this is the first time someone has
+  // installed the add-on, so we can wait for previews to come back
+  simpleStorage.storage.waitForFirstPreviews = {};
```

Comment:
`simplePrefs` is already included in `lib/ActivityStreams.js`, and we can have a default value in package.json.
We could have two prefs: one for top_frecent and another for highlights_links.

The code here would blow away the `storage` every time you boot up the addon it looks. You want a more persistent pref, right?